### DISCUSSION TITLE
如果城市裡到處都是電池，還需要害怕停電嗎？

### DISCUSSION THREAD

**Discussion by leon10456** [1101-GS4719-C/Python-Community-GS4719-C#500] [General]
最近停電的事情鬧得沸沸揚揚 那如果將城市設施裝滿電池 搭配上電動機車要換電池 維持電池循環及電池健康度 是否能在停電時維持城市設施運作呢
https://technews.tw/2022/03/18/kymco-ionex-at-kaohsiung-smart-city/

**Comment by ThomasMeng0515** [comment]
這樣電池的伏特數、容量就要依照各個應用去調整，去設計，感覺很麻煩

**Comment by chupochun** [comment]
如果製造太多電池對環境會不會有影響

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，期望未來能夠永不缺電。

**Comment by sean1325418** [comment]
還是要考慮成本，花大錢裝這麼多，可是一年只用一、兩次這樣符不符合機會成本。

**Comment by Sazmafuji** [comment]
前天我家wii的遙控器電池漏液，害我沒遙控器玩，我認為裝滿電池，可能會使城市都受到極大汙染，但如果可解決，此方法應該會是未來對付停電的好方法

**Comment by hapiwangy** [comment]
也許不失為一個可行的方案，但也許比較過機會成本之後可能願意花錢執行的人並不多

**Comment by lx3211** [comment]
電池如果沒有好好回收會對環境產生極大的傷害

**Comment by PoooYen** [comment]
個人認為建設成本將會是非常大的一個問題

**Comment by JSHT** [comment]
覺得外部成本會遠大於實質效益。

**Comment by 11timyyy** [comment]
計畫容易，但實際建設可能又會有困難

**Comment by yuchi06274** [comment]
如果將來實施 電池需要有處理管道 才不會對環境造成極大的危害

**Comment by yucheng1028** [comment]
感覺電池汙染會讓這根本沒辦法執行，而且現在台灣缺的不是電，是愛，還有台灣價值。

**Reply by yi-1111** [reply]
一是五零很強大，我們絕對不缺電

**Comment by ghost** [comment]
想多了，真這樣還得思考廢棄物如何處理

**Comment by dayu1004** [comment]
聽起來很夢幻 但實際上要實施好像還有一點差距哈哈（雖然好像不太可能實施）

**Comment by HuanYuuuuuu** [comment]
像GOGORO的電池約可以行駛 5萬公里，電池大概就報廢了。
當有大量電池廢棄物時更不可能要求所有人都將廢棄電池丟到該丟的地方，到時電池廢棄物造成的環境汙染想必也會很可怕

**Comment by xkllkx** [comment]
電池畢竟不是一個很環保的產物，目前缺電是因為發出來的電需要馬上使用，個人覺得抽蓄發電是個不錯的概念。

**Comment by stan890425** [comment]
我們現在的店大多都線發現用，有了這個就不怕停電了~~

**Comment by jcpan821** [comment]
不太有可能弄出足以供應整座城市的電池吧

**Comment by alexthebigpp** [comment]
但製造電池花的成本可能更高。

**Comment by Ben6uo** [comment]
如果發生意外的話 後果頂不著阿

**Comment by etin2468** [comment]
電池也是高污染的產品，我覺得對環境有很大的影響ㄟ

**Comment by bloodtrail0511** [comment]
如果有了這個就不怕停電了

**Comment by Lynn12378** [comment]
感覺應該從根本去處理停電，電池的設置還牽扯到位置、密度、人力和成本等。
